lib/parser.dart

```diff
@@ -488,7 +488,7 @@ class _Parser {
           _error('missing import string', _peekToken.span);
         }
 
-        return ImportDirective(importStr.trim(), medias, _makeSpan(start));
+        return ImportDirective(importStr!.trim(), medias, _makeSpan(start));
```

Comment:
yeah, I _really_ wanted to be able to rely on that, but `_error` doesn't throw, it collects errors in a list and then lets things keep going for some reason. (Or at least it does that indirectly through the `Messages` object and a configurable `printHandler` which _doesn't_ print...)

My guess is this is a code path that's never been hit since it would have failed before right? I'm trying to avoid doing too much to fix these types of bugs because I'm afraid it'll turn into a huge refactor and likely a breaking change.

Reply:
kk